src/services/ui/src/pages/Home.tsx

```diff
@@ -1,3 +1,142 @@
+import {
+  ActionsMenu,
+  Button,
+  Header,
+  Link,
+  Logo,
+  Footer,
+  UsaBanner,
+} from "@enterprise-cmcs/macpro-ux-lib";
+import cmsLogo from "@enterprise-cmcs/macpro-ux-lib/build/assets/img/logos/cms_logo.svg";
 export const Home = () => {
-  return <>Home Page</>;
+  return (
+    <>
+      <Header
+        headerLogo={
+          <Link href="" title="Project Title">
+            <Logo altText="Project Logo" source={cmsLogo} />
+          </Link>
+        }
+        navData={[
+          {
+            buttonText: "Current Section",
+            columns: [
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
+              [
+                {
+                  href: "",
+                  text: "Navigational Link",
+                },
+              ],
```

Comment:
im guessing you just copy pastad from the storybook example. but if these links dont go anywhere only include maybe one.

Reply:
That's fair. Good point Ben. Yeah if we can really slim this down it would be nice.